README.md

```diff
@@ -83,7 +83,7 @@ Lark is great at handling ambiguity. Here is the result of parsing the phrase "f
 
 ![fruitflies.png](examples/fruitflies.png)
 
-[Read the code here](https://github.com/lark-parser/lark/tree/master/examples/fruitflies.py), and [more examples here](https://github.com/lark-parser/lark/tree/master/examples)
+[Read the code here](https://github.com/lark-parser/lark/tree/master/examples/fruitflies.py), and [more examples here](https://github.com/lark-parser/lark/tree/master/examples) and [here](https://github.com/ligurio/lark-grammars).
```

Comment:
I am not sure if this is the best place to add this, since the other links a complete python programs, where as this is more a collection of grammars.

Reply:
What is a better place for it?

Reply:
I am not sure. @erezsh Opinion on this would also be appreciated.

- Further down in the list of projects.
- An additional `### Header` above/below the projects marking this as a library of grammars.

Reply:
Considering that the grammars have no tests, and some of them aren't exactly idiomatic, I'm reluctant to allow them into the README.

We can place a link to them in the docs, for example [in this page](https://lark-parser.readthedocs.io/en/latest), and just make it clear that they are external examples.

Reply:
Moved a link to a list with resources in doc index with name "Third-party examples".